pkg/manifests/manifests.go

```diff
@@ -2926,7 +2926,7 @@ func (f *Factory) MonitoringPluginDeployment(tlsSecret *v1.Secret) (*appsv1.Depl
 	// Hash the TLS secret and propagate it as an annotation to the
 	// deployment's pods to trigger a new rollout when the TLS certificate/key
 	// are rotated.
-	d.Spec.Template.Annotations["monitoring.openshift.io/hash"] = hashByteMap(tlsSecret.Data)
+	d.Spec.Template.Annotations["monitoring.openshift.io/monitoring-plugin-cert-hash"] = hashByteMap(tlsSecret.Data)
```

Comment:
CMO should handle this change well: https://github.com/openshift/cluster-monitoring-operator/blob/60312aa5cb3d27d9c4134c55a1f4d0116bf8e71d/pkg/client/client.go#L1972-L1991

Reply:
nit-picking but `monitoring-plugin-cert-hash` is a bit verbose since the annotation is set on the monitoring-plugin deployment. `cert-hash` would be more adequate.

Reply:
was trying to stick to `RESOURCE-NAME-hash`, but sure, let's save some bytes.